code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -286,3 +286,62 @@
 	desc = "A plasmaman envirohelm designed by Space Cola Co for the plasmamen."
 	icon_state = "coke_envirohelm"
 	item_state = "coke_envirohelm"
+
+/obj/item/clothing/head/helmet/space/plasmaman/tacticool
+	name = "diver envirosuit helmet"
+	desc = "A plasmaman helm resembling old diver helms."
```

Comment:
![image](https://github.com/user-attachments/assets/ce89fdc1-6f68-46c3-8b70-3d67039d2372)

Could indicate that it can be reskinned here, just like the clown mask does for example.